### DISCUSSION TITLE
Dataset RT.cvs

### DISCUSSION THREAD

**Discussion by Ines1306** [dssg-pt/covid19pt-data#721] [Amostras]
Boa tarde.

O dataset RT.cvs tem dados de RT a partir de 21/02/2020 com valores acima de 0, no entanto os primeiros casos em covid em PT apenas surgiram a 02/03/2020.

**Comment by davipt** [comment]
Olá Inês.

O dataset rt.csv é a transformação dos dados oficiais de múltiplos ficheiros Excel disponíveis no site oficial em [insa.min-saude.pt](http://www.insa.min-saude.pt/category/areas-de-atuacao/epidemiologia/covid-19-curva-epidemica-e-parametros-de-transmissibilidade/) para um csv único em formato wide-table e com as regiões todas juntas.

O Rt é calculado usando a data de início de sintomas, que é diferente da data de notificação no sistema.
A informação de inicio de sintomas não é pública, mas está disponível para analistas, e está encapsulada no gráfico do boletim diário.

No [primeiro relatório público](http://www.insa.min-saude.pt/wp-content/uploads/2020/05/Report_covid19_28_05_2020.pdf) é mais fácil ver o gráfico e verificar que houve casos logo a 16 de Fevereiro (ou nessa semana), embora só tenham sido notificados no sistema a 2 de Março. Essa semana de casos é a usada para calcular o primeiro valor de Rt a 22 de Fevereiro.

Faz sentido isto?

Obrigado

**Comment by nunodevops** [comment]
Olá,

Em primeiro lugar, muito obrigado pelo vosso trabalho.
Já há algum tempo que utilizo os vossos dados numa página que criei no meu website. Há algum tempo que tenho verificado que o ficheiro rt.csv deixou de ter dados atualizado e que agora existe no ficheiro data.csv dois campos com o rt_nacional e rt_continente, isto quer dizer que o ficheiro rt.csv vai ser descontinuado? Qual devemos utilizar? Interessa-me principalmente o valor de rt_nacional.

Se descontinuarem o ficheiro rt.csv, será que poderiam migrar os dados do ficheiro rt.csv para o data.csv é que no data.csv os dados sobre o rt só começam a 15-03-2021, se pretenderem poderei ajudar nisto.

1Abraço.

**Reply by davipt** [reply]
Olá Nuno. 

Os valores no data.csv são a incidência e RT que estão no PDF do relatório diário. O site do ministério da saúde com os relatórios está temporariamente indisponível mas há cópia de todos os relatórios na pasta dgs-*. Estes valores existem desde que foi adicionado o quadradinho da incidência vs. Rt no relatório. 

O Rt.csv é a conversão do ficheiro excel do site da INSA https://www.insa.min-saude.pt/category/areas-de-atuacao/epidemiologia/covid-19-curva-epidemica-e-parametros-de-transmissibilidade/ e é normal ter uns dias de atraso pois o relatório é semanal. O processo é automático portanto assim que actualizem os ficheiros no site, será também actualizado aqui. 

Os Rt da INSA são processados com os dados disponíveis e são actualizados retroactivamente. São os correctos. O Rt diário do relatório é aproximado. 

Vejo que o último update foi há seis dias e que só tem dados até dia 2. Veremos amanhã se actualizam os dados em falta no site da INSA. 

Obrigado pelo aviso e até amanhã.

**Reply by davipt** [reply]
O Rt foi actualizado automaticamente há 19 horas com dados até dia 9 e em conjunto com o relatório em PDF de dia 14 https://www.insa.min-saude.pt/category/areas-de-atuacao/epidemiologia/covid-19-curva-epidemica-e-parametros-de-transmissibilidade/
